src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/table.scss

```diff
@@ -78,4 +108,50 @@ $cellBackgroundColorActive: $shakespeare;
         border-radius: 0 3px 3px 0;
         border-right-width: 1px;
     }
+
+    &.small {
+        width: 1%;
+    }
+}
+
+.control-cell {
+    padding: 0;
+    background-color: $cellBackgroundColor;
+    color: $cellTextColorActive;
+    text-align: center;
+    width: 1%;
+
+    &:hover {
+        button {
+            visibility: visible;
+            background-color: $cellBackgroundColorActive;
+            border-right: 1px solid $controlCellBorderColor;
+        }
+    }
+
+    button {
+        visibility: hidden;
+        width: 55px;
+        height: 38px;
+        border: none;
+        color: inherit;
+        cursor: pointer;
+        background-color: transparent;
+    }
+}
+
+.table-placeholder-area {
```

Comment:
Why don't we have a `table-placeholder` class with nested `area`, `icon` and `text` classes, instead of three prefixed classes?

Reply:
nesting increases the specificity of a selector. We should only use that for state specific selectors (e.g. small, hidden, etc.).